Enable Celo buyback from FeeHandler and fee distributions (#360)
celo-org/governance: CGPs/cgp-0108.md

---
cgp: '0108'

> title: 'Activate buyback of non-Celo fees and fee distribution'

Author: ```suggestion
title: 'Enable Celo buyback from FeeHandler and fee distributions'
```
